src/bot/lib/matching.js

```diff
@@ -57,10 +57,28 @@ export function prepareMatchOutput(matchResults) {
   };
 }
 
+// processes matches for additions and returns output for dialogue
+export function prepareAddOutput(matchResults, token) {
+  const {matches, match, exact} = matchResults;
+  const output = [];
+
+  // suggest list if there are matches
+  if (matches.length !== 0) {
+    output.push(`_Try using the \`list\` command to see what items already exist._`);
```

Comment:
Following the current pattern, shouldn't this throw with that message if a match was found?